docs/dmytro-shevchenko1/GamesProject/calculator.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="./styles/style.css">
+        <link rel="stylesheet" type="text/css" href="./styles/header.css">
+        <link rel="stylesheet" type="text/css" href="./styles/calculator.css">
+        <link rel="stylesheet" type="text/css" href="styles/footer.css">
+        <title>Calculator</title>
+    </head>
+    <body class="calculator-page">
+        <header class="header">
+            <div class="container">
+                <div class="header-flex">
+                    <img src="images/logo.svg" alt="Logo">
+                    <div class="nav">
+                        <ul class="nav-items">
```

Comment:
Hey, @dmytro-shevchenko1, I see a content jumps for different values
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/bd8ca939-6b43-48f3-8c2f-8147f7bb7de0)
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/fd9911b1-f0fc-44c7-ab82-277618f16222)

Reply:
And by the design, the calculator should be aligned to the left
<img width="750" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/6ff00589-e6a1-4c87-b9ef-1b1544c7857e">